samples/net/wifi/README.rst

```diff
@@ -0,0 +1,50 @@
+.. _wifi:
+
+Wi-Fi sample
+############
+
+Overview
+********
+
+This sample enables testing Wi-Fi drivers for various boards.
+It simply enables the Wi-Fi shell module that will provide a set
+of commands: scan, connect and disconnect.
+It also enables net shell module to verify net if settings.
+
+Building and Running
+********************
+
+Make sure the board and the chip you are targeting are supported for
+this sample.
+
+For instance, Atmel's Winc1500 chip is supported on top of
+quark_se_c1000_devboard board.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/wifi
+   :board: quark_se_c1000_devboard
+   :goals: build
+   :compact:
+
+Sample console interaction
+==========================
+
+.. code-block:: console
+
+shell> select wifi
```

Comment:
the contents of this code block need to be indented to the ``c`` in ``code-block`` (add three leading spaces to lines 35-50)